resources/sass/pages/_theme-detail.scss

```diff
@@ -0,0 +1,281 @@
+.ris-theme-detail {
+    &__text {
+        -webkit-box-orient: vertical;
+        display: -webkit-box;
+        line-height: $lh-l;
+        overflow: hidden;
+
+        @include breakpoint($bp-tablet-min) {
+            -webkit-line-clamp: 20;
+            max-height: $lh-l * 20;
+        }
+
+        @include breakpoint($bp-mobile-max) {
+            -webkit-line-clamp: 40;
+            max-height: $lh-l * 40;
+        }
+    }
+
+    .ris-section-wrapper {
+        &:last-child {
+            border-bottom: 0;
+        }
+
+        &_full {
+            padding-left: 0;
+            padding-right: 0;
+
+            @include breakpoint($bp-tablet-min) {
+                padding-top: $indent-s;
+            }
+
+            .ris-headline,
+            .ris-caption {
+                @include breakpoint($bp-mobile-max) {
+                    padding-left: $indent-m;
+                    padding-right: $indent-m;
+                }
+            }
+        }
+    }
+
+    .ris-title {
+        font-size: $fs-l;
+        line-height: $lh-m;
+        margin-bottom: $indent-s;
+    }
+
+    .ris-h2 {
+        margin-bottom: $indent-m;
+    }
+
+    .ris-img-wrapper {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: $width-section-1024;
+        width: 100%;
+
+        &:not(:last-child) {
+            margin-bottom: $indent-m;
+        }
+
+        .ris-link {
+            width: 100%;
+        }
+
+        .ris-img {
+            width: 100%;
+        }
+    }
+
+    .ris-caption {
+        margin-top: $indent-s;
+
+        &:not(:last-child) {
+            margin-bottom: $indent-s;
+        }
+    }
+
+    .ris-map-detail {
+        @include breakpoint($bp-tablet-min) {
+            display: flex;
+            justify-content: center;
+        }
+
+        .ris-img {
+            border-radius: $b-radius-primary;
+        }
+
+        .ris-link {
+            @include breakpoint($bp-tablet-min) {
+                width: $width-interactive-tablet-300;
+            }
+        }
+    }
+
+    .ris-map-all-street-address {
+        display: flex;
+
+        @include breakpoint($bp-mobile-max) {
+            margin-bottom: $indent-m;
+        }
+
+        &::before {
```

Comment:
do you really need to define icon on the element itself?

Reply:
Fixed